tests/test_persistence.py

```diff
@@ -977,6 +993,14 @@ def test_to_json(default_conf, fee):
                       'strategy': None,
                       'timeframe': None,
                       'exchange': 'binance',
+
```

Comment:
As this does not add the trade to a session, defaults won't apply (that's when sqlalchemy applies defaults).

to test this you can however add "leverage" to the trade instance above - and it should "get dumped out" down here ...